### ISSUE TITLE
Recycle - Force option shouldn't delete permanently

### ISSUE THREAD

**Issue by akaleeroy** [bdukes/PowerShellModules#18]
`Remove-ItemSafely -Force` becomes a misnomer if the `-Force` switch makes the command unsafe (delete permanently, without a confirmation prompt). For deleting permanently users ought to use `Remove-Item`.

Maybe a change of name of the cmdlet?

**Comment by bdukes** [comment]
Yes, the cmdlet is poorly designed, in that using `Remove-Item` parameters that aren't supported causes it to fall back to `Remove-Item` (and, therefore, silently before "unsafe"). Fixes are definitely welcome.
